packages/docs/apis/props.md

```
@@ -2,8 +2,9 @@
 
 | 名称                | 类型                                                                                  | 默认值             | 说明                                                                                                                                                                                                           |
 | ------------------- | ------------------------------------------------------------------------------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| default-field       | string                                                                                | --                 | 自定义按下 enter 键时，默认搜索的范围；此属性为空时。则默认在关键字范围下搜索 xxx，即生成的搜索标签为“关键字：xxx”                                                                                             |
-| editable            | boolean                                                                               | false              | 是否开启标签编辑功能，（注：map 类型不支持编辑）                                                                                                                                                               |
+| default-field       | string                                                                                | --                 | 自定义按下 enter 键时，默认搜索的范围；此属性为空时。则默认在关键字范围下搜索 xxx，即生成的搜索标签为“关键字：xxx”                                                                                            |
+| default-field-replace | boolean                                                                   | false                 | 自定义按下 enter 键时，默认搜索的范围；此属性为true时。则默认替换上一个搜索条件   
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)